### ISSUE TITLE
9. Images, Art & Video - [E2] 2. Wang, Yilun, and Michal Kosinski. 2018.

### ISSUE THREAD

**Issue by JunsolKim** [UChicago-Computational-Content-Analysis/Readings-Responses-2023#7]
Post questions here for this week's exemplary readings: 2. Wang, Yilun, and Michal Kosinski. 2018. “Deep neural networks are more accurate than humans at detecting sexual orientation from facial images.” Journal of personality and social psychology 114(2): 246.

**Comment by konratp** [comment]
Even as someone who was very little professional or academic knowledge about the technologies behind facial recognition, I understand that these technologies often operate in societal frameworks that significantly favor members of dominant majority groups over minorities. My understanding so far has been that many issues arise from AI technologies being worse at detecting, for example, a non-white person's facial features than those of a white person, resulting in malfunctions that can have severe consequences. 

However, prior to reading this paper, I had not considered that minority rights might be endangered precisely by the accuracy of these methods. Institutional homo- and transphobia is rather common in many places around the world, so I believe that the findings of this paper -- as the authors themselves contend -- are rather impactful and important for queer people and organizers to understand. While the authors sound the alarm in the paper and alert readers of the important implications of their findings, I wish they had also provided us readers with some more comforting news, for example, potential ways in which the scientific community can counteract such trends that allow governments to use these technologies to discriminate against minorities. What are some policies or technological developments that could counteract these trends?

**Comment by ValAlvernUChic** [comment]
This paper undoubtedly brings up a ton of ethical questions - one of the most important harkening back to the question on the potential downsides of GPT. Here, researchers claim they were able to identify homosexual individuals from their faces alone, using only computer vision. I can imagine the same things eventually being measured at the same level of accuracy for text. It's already completely possible to identify the identity of a person based on their writing - can't imagine that doing the same for particular identities is too far off.

**Comment by Jasmine97Huang** [comment]
This article is no stranger for 2nd year students in the MACS program, as we had the pleasure attending the workshop where one of the authors presented. Some concerns are around the privacy and potential abuse of such technology. Would love to have more discussions on this.

**Comment by pranathiiyer** [comment]
To add onto Val's point, there's so much that can be done with image data but there are also several questions of ethics that come into play when using them for purposes such as those mentioned in these paper and research that aims to identify convicts based on visual cues. Do we need to and can we automate highly subjective and complex processes such as these?

**Comment by NaiyuJ** [comment]
This article is worth discussing on two levels of ethical concerns. The first is the ethics of researching LGBT communities and issues, that is, whether the research question is worth asking and adds to existing knowledge. The second is the ethics of research methods, that is, whether using a large scale of online personal information and machine learning techniques is suitable to study the sensitive topic.

**Comment by shosamane** [comment]
I was wondering if any of the ethical research requiring human subjects for training had already started using synthetic data generation models like GANs. I believe this could at least help validate the stated hypothesis (if someone is trying to predict based on facial features, again, for ethical purposes) albeit not generalise it.

**Comment by Jiayu-Kang** [comment]
I'm skeptical about the authors claim that this study can "advance our understanding of the origins of sexual orientation." To what extent can researchers actually infer or even draw conclusions on the nature or mechanism of some observed phenomenon based on a classification model?

**Comment by Qiuyu-Li** [comment]
This is actually a paper that occurred in one of my first quarter’s courses Perspectives on Computational Analysis. I thought, and still think this paper is unethical. One of the biggest concerns here is that homosexual people may be discouraged to share their personal information on social media or to talk about their sexual orientation. The reasons are two-fold: sexual minority people may feel unsafe if they are aware that their private data is being used in such a public way, and they may also be frightened from sharing their photos given the unsettling conclusions drawn by Wang and Kosinski (2018). I would love to hear more about ethical-related discussions about computational social science studies with image and video data.

**Comment by LuZhang0128** [comment]
I've read a similar, if not the same, article in the workshop last quarter. It is not hard to imagine a heated debate on ethical questions due to the conclusion. I wonder if future research can focus on developing algorithms to protect these vulnerable communities instead of exposing them to greater harm.

**Comment by mikepackard415** [comment]
This paper always produces a lot of ethical concerns. It even got a little tense when Kosinski came to the workshop last year. I do think the authors intentions were and are good, but I also think it is worth talking about whether this is the kind of research that is worth pursuing.

**Comment by hshi420** [comment]
I'm interested in the interpretability of these GAN models. Are they more interpretable than other neural network models? Are any of these black box 'blacker' than others?

**Comment by Emily-fyeh** [comment]
Similar thoughts to most of my peers, I think knowing what the GAN generation can do does not mean that it is legitimate to spread the unethical use of it. But still, I would want to see if this kind of stereotyping/categorizing people based on sexual orientation listens more to LGBT people.

**Comment by ttsujikawa** [comment]
As many of second year MACSS students say, this paper raised a lot of ethical concerns while some interpretations are very interesting. The capability of GAN models is highly powerful, but we should always be careful not only about what we can do with that but about how we should ethically utilize that.

**Comment by jacyanthis** [comment]
This is a good follow-up on this paper by a Northwestern student forthcoming in _JPSP_:

[https://psyarxiv.com/u7vcd](https://psyarxiv.com/u7vcd)

"This article tests whether people of different sexual orientations presented themselves distinctively in photographs, and whether these distinctions revealed their sexual orientation. I found significant differences in self-presentation. For example, gay individuals were on average more likely to wear glasses compared to heterosexual individuals in images uploaded to the dating website. Gay men also uploaded brighter images compared to heterosexual men. To further test how some of these differences drove the classification of sexual orientation, I employed image augmentation or modification techniques. To evaluate whether the image background contributed to classifications, I progressively masked images until only a thin border of image background remained in each facial image. I found that even these pixels classified sexual orientations at rates significantly higher than random chance. I also blurred images, and found that merely three numbers representing the brightness of each color channel classified sexual orientations"

**Comment by sizhenf** [comment]
Apart from the ethical debate, I'm curious about if plastic surgeries or cosmetic treatments will effect the accuracy of the method.

**Comment by AllisonXiong** [comment]
My question would be the comparison task used for computer and human judges. That is, judges were asked to pick the gay face from a pair of homo-heterosexual pair. I would think this kind of tasks led to ranking of all faces, and since the distribution of the sample data is extremely different from the reality, is this ranking problematic when applied to real world setting?

**Comment by chuqingzhao** [comment]
As this paper has already been discussed in our first-year perspective class and it has received a series of debates on the ethnical issues of using deep learning tools, I am wondering when applying the "black-box" algorithms what's kind of ethnical concerns should be paid more attention to?
